azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/error/CallCompositeError.kt

```diff
@@ -5,4 +5,7 @@ package com.azure.android.communication.ui.error
 
 import com.azure.android.communication.ui.configuration.events.CallCompositeErrorCode
 
-internal data class CallCompositeError(var callCompositeErrorCode: CallCompositeErrorCode, var cause: Throwable,)
```

Comment:
Reving data class will not be supported by MutableStateFlow.

Reply:
data classes are compared 'by value' it results in:
have an error with particular error code
process this error
then if there is a new error occurred with same error code 
then error manager (that is tracking last processed error) compares a new instance of the new error and finds it equal to already processed error.